site/src/pages/_app.page.tsx

```diff
@@ -20,6 +20,7 @@ import {
   UserContextValue,
   UserState,
 } from "../context/user-context";
+import { defaultSeoConfig } from "../default-seo-config";
```

Comment:
It's helpful to keep things that are used together close to each other. Extracting a separate file makes sense if we plan to use `defaultSeoConfig` outside `_app.page.tsx`. I suggest defining this constant right in the file or even ditch the constant and directly set `<DefaultSeo ... />` props.

I may be missing something though. What benefits of a separate file to you see?

Reply:
I wanted to keep `_app.page.tsx` clean & easy-to-read. Thought keeping this SEO content on somewhere else.

Reply:
Moved the props into the top of `_app.page.tsx` here https://github.com/blockprotocol/blockprotocol/pull/562/commits/1c2fe0d3010dacbd8f846be8aa1514d109383e5a